src/Pages/Jobs/JobCard.jsx

```diff
@@ -33,6 +34,10 @@ const JobCard = ({ job, index }) => {
         return () => clearTimeout(timer);
     }, [index]);
 
+    const url = `https://code-routine.vercel.app/jobs/${job._id}`;
+    const title = "Check out this amazing page!";
+    const description = "This is an awesome page that you should definitely check out.";
+
```

Comment:
why these are line?

Reply:
kept it for future use, but removed it as of now